src/components/Project/FeedbackModule.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Section from '../General/Section';
+import LongCard from '../General/LongCard';
+import styled from 'styled-components';
+import {breakpoint} from '@aragon/ui';
+const medium = css => breakpoint('medium', css);
+const large = css => breakpoint('large', css);
+import speech from './assets/speech.svg';
+
+const FeedbackModule = () => (
+  <StyledSection>
+    <Container>
+      <Feedback>
+        <ImageContainer>
+          <img src={speech} />
```

Comment:
You'll probably need to add an `alt` attribute here (in case React did not show you a warning)